src/Microsoft.AspNet.SignalR.SqlServer/SqlScaleoutConfiguration.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.SignalR.Messaging;
+
+namespace Microsoft.AspNet.SignalR
+{
+    /// <summary>
+    /// Settings for the SQL Server scale-out message bus implementation.
+    /// </summary>
+    public class SqlScaleoutConfiguration : ScaleoutConfiguration
```

Comment:
Currently ScaloutConfiguration is in Microsoft.AspNet.SignalR.Server, after refactor,  ScaloutConfiguration will be ScaloutOptions  in Microsoft.AspNet.SignalR.Messaging, so I thought to change SqlScaleoutConfiguration  after refactor.